packages/widgets/src/components/Pagination/PaginationContext.tsx

```diff
@@ -0,0 +1,22 @@
+import { createContext, useState, useContext } from 'react';
+import { IPaginationContext, IPaginationContextProviderProps } from './types';
+
+export const PaginationContext = createContext<IPaginationContext>({
+  setPage() {
+    /* no-op */
```

Comment:
Do we need defaults for the createContext? It could be bring to the uncovered test case

Reply:
Yeah, unfortunately, all the required properties should have defaults. I tried adding a comment `istanbul ignore next` but that didn't work. 
And yes, that decreases the coverage %.

Reply:
I think, we can try without defaults, the empty object, but using `as` to specify interface.
[I've already tried this approach](https://github.com/ssagroup/ui-kit/pull/68/files#diff-23813e7e0c992c83cd826896e51b5da380ded573a4ecce2a979f82262d8dbe60R5)